<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>每秒复制源文件一行到目标文件</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_sourcePath"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_selectTargetFile">
     <property name="text">
      <string>选择目标文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_targetPath"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_selectSourceFile">
     <property name="text">
      <string>选择源文件</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="textEdit_log"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>日志</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="btn_runCopy">
     <property name="text">
      <string>开始复制</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
